src/components/Input.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Input extends Component {
+  onChangeInputHandler = event => this.props.onChangeHandler(event)
```

Comment:
I think, that this is not needed here. You could make simply:
`onChange={this.props.onChangeHandler}` event will be passed as a argument